src/main/java/carracing/domain/car/Car.java

```diff
@@ -23,6 +23,16 @@ public Integer getPosition() {
         return this.position.getPosition();
     }
 
+    @Override
+    public int compareTo(Car o) {
```

Comment:
지금 비교하는걸보면 Position을 기준으로 비교하고 있습니다. 그럼
이 역시 비교의 책임을 Position에게 전달할 수 있어요 ㅎㅎ 

```suggestion
    public int compareTo(Car o) {
        return position.comparTo(o.getPosition());
```

그리고 Position객체 내부적으로는
```java
return position - o.position;
```

이정도로만 표현해도 충분할꺼같네요!